availability/tests/unit/test_config.py

```diff
@@ -16,7 +16,7 @@
 import mock
 
 from availability import config
-from tests.unit import test
+from availability.tests.unit import test
```

Comment:
Tests module is something separate, it is not a part of availability module. I would request keeping tests module in old place (package root)